packages/formation-react/src/components/PromoBanner/PromoBanner.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+const PROMO_BANNER_TYPES = {
```

Comment:
Is there value in limiting the banner types? Or could we let people pass in the icon as a prop?

Reply:
I considered having a `customIcon` prop, but the requirements seemed strict about having only three types of these so I decided to make the code reflect that. I'm fine with adding the `customIcon` prop though.

Reply:
I don't have a strong opinion, if we're trying to limit the types, that totally makes sense.